test/Microsoft.AspNetCore.Mvc.FunctionalTests/GlobalAuthorizationFilterTest.cs

```diff
@@ -51,10 +55,15 @@ public GlobalAuthorizationFilterTest(MvcTestFixture<SecurityWebSite.StartupWithG
         }
 
         [Fact]
-        public async Task AuthorizationPoliciesDoNotCombineByDefault()
+        public async Task AuthorizationPoliciesDoNotCombine_WithV2_0()
```

Comment:
Kept tests covering `2_0` and `2_1` authorization policy behaviours because both were covered before. But, no longer need separate `CombineAuthorizeTests` or `StartupWithGlobalAuthorizeAndAllowCombiningAuthorizeFilters`.